packages/fast-components-styles-msft/src/typography/index.ts

```diff
@@ -2,18 +2,18 @@ import { DesignSystem } from "../design-system";
 import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
 import { TypographyClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 import { applyTypeRampConfig } from "../utilities/typography";
-import { ensureForegroundNormal } from "../utilities/colors";
-import { toPx } from "@microsoft/fast-jss-utilities";
-import { get } from "lodash-es";
+import { neutralForegroundRest } from "../utilities/color";
 
 const paragraphDefaults: CSSRules<DesignSystem> = {
-    color: ensureForegroundNormal,
+    color: neutralForegroundRest,
     marginTop: "0",
     marginBottom: "0",
 };
 
 const styles: ComponentStyles<TypographyClassNameContract, DesignSystem> = {
-    typography: {},
+    typography: {
+        transition: "all 0.2s ease-in-out",
```

Comment:
Doesn't respond to state interaction.

Reply:
Jason, Chris and I discussed this yesterday. Theme change can happen on any of our components on the devsite and since there is real no performance hit we decided to include transitions on all components.

Reply:
Sounds good.